### DISCUSSION TITLE
Is there a way to load a picture onto the NSPanel display?

### DISCUSSION THREAD

**Discussion by st2000** [Blackymas/NSPanel_HA_Blueprint#2131] [Q&A]
I am new to ESPNow and Home Assistant. So I'm not even sure what to search for. In short, when someone rings the door bell I want to push an image from the outside camera to the NSPanel display.

**Comment by edwardtfn** [comment]
The short answer is "this is NOT possible", but you will find more details on #1270

**Reply by st2000** [reply]
Wow, that's exactly what I wanted to do! 
(For an instant, I thought I had double posted!) 
Thanks for responding to my question even though is has already been asked.
